index.js

```diff
@@ -0,0 +1,58 @@
+const students = [
+    {
+        name : "Peyton",
+        grade : 47
+    },
+    {
+        name : "Emily",
+        grade : 56
+    },
+    {
+        name : "Claus",
+        grade : 35
+    },
+    {
+        name : "Nelly",
+        grade : 72
+    },
+    {
+        name : "Brandon",
+        grade : 80
+    },
+    {
+        name : "Trent",
+        grade : 79
+    },
+    {
+        name : "Chris",
+        grade : 61
+    },
+    {
+        name : "James",
+        grade : 60
+    },
+    {
+        name : "Alex",
+        grade : 86
+    },
+    {
+        name : "Khloe",
+        grade : 97
+    }
+]
+
+function getPassingStudents (arr){
+
+    let passingStudents = [];
+
+    for (let i = 0; i < arr.length; i++){
+
+        if (arr[i].grade >= 60){
```

Comment:
60 here is what we call a magic number.
The number represents something that is not obvious by looking at the code.
So I would extract it to a variable explaining what it's for.
Create a constant outside the function called passingScore or similar.